client/electron/snap/snapcraft.yaml

```diff
@@ -55,6 +55,8 @@ parts:
       - -libparsec.node
     build-attributes:
       - enable-patchelf
+    build-environment:
+      - LIBPARSEC_FORCE_ZSTD: real
```

Comment:
This is not needed since on linux the real zstd is always used

Reply:
Better safe than sorry